clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -154,7 +154,7 @@ scheduler_impl_tmpl_t<memref_t, reader_t>::get_reader(const std::string &path,
         }
         for (; iter != end; ++iter) {
             const std::string fname = *iter;
-            if (fname == "." || fname == ".." ||
+            if (fname == "." || fname == ".." || fname == DRMEMTRACE_AUX_DIRNAME ||
```

Comment:
Wondering if more files should now be removed from these hard-coded ignore lists and put under `aux` (e.g., `v2p.textproto`).
Or if we should leave it like this for now.

Reply:
Yeah I had the same question. Moving would be a compatibility-breaking change as existing files in the v2 traces dir would need to be moved too. A middle ground that keeps backward-compatibility would be to keep logic to allow existing files in the trace dir, but also check for them in `aux`.

Adding `aux/` for only new files is simpler though.

@derekbruening FYI

Reply:
It seems weird to me to have some metadata files in the base dir and some in aux/.  I think we should move them all to aux/.  We're bumping the version anyway?  The minor version I guess.  This doesn't seem a major compatibilty issue: minor enough the minor bump is enough.

Reply:
I wonder if the trace files themselves should be in their own dir too.  Or, we already have raw/ and trace/: should aux/ be a peer of those instead of inside trace/ and we leverage our support for passing the parent of raw/,trace/?  Though that complicates other uses where trace/ is preserved/moved and raw/ is discarded?

Reply:
> We're bumping the version anyway? The minor version I guess.

Compatibility issues due to a changed directory structure will manifest before the tools have had a chance to look at the trace version and change their expectations.

Reply:
If we're okay with bigger compatibility breaks, we could potentially take this opportunity to make the info.textproto more central to our tools. We've discussed previously (offline perhaps) about providing some top-level trace info in some file, which could hold many other details and make them more readily available (collection time, trace version, etc.)